src/bluecadet/core/SettingsManager.cpp

```diff
@@ -114,8 +122,8 @@ void SettingsManager::parseJson(ci::JsonTree & json) {
 	setFieldFromJsonIfExists(&mBorderless, "settings.window.borderless");
 	setFieldFromJsonIfExists(&mWindowSize.x, "settings.window.size.x");
 	setFieldFromJsonIfExists(&mWindowSize.y, "settings.window.size.y");
-	setFieldFromJsonIfExists(&mWindowPos.x, "settings.window.position.x");
```

Comment:
Just confirming: `window.position` wasn't being used before, right? So it's safe to rename to `pos`?

Reply:
Yeah, I couldn't find any legacy code that referenced it, but I admit I didn't dig around that deep.

Reply:
Cool!